content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,79 @@ description: Istio 1.5 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
+
+- **Improved** performance of `ServiceEntry` updates #19305.
+- **Improved** Envoy sidecar readiness probe to more accurate determine readiness #18164.
+- **Improved** performance of configuration updates by sending partial updates where possible #18354.
+- **Added** an option to control locality load balancer settings for each Service #18406.
+- **Fixed** an issue where pods crashing would trigger excessive configuration pushes #18574.
+- **Fixed** issues with applications that call themselves #19308.
+- **Added** detection of `iptables` failure when using Istio CNI #19534.
+- **Added** `consecutive_5xx` and `gateway_errors` as outlier detection options #19771.
+- **Improved** `EnvoyFilter` matching performance #19786.
```

Comment:
```suggestion
- **Improved** `EnvoyFilter` Envoy proxy matching performance #19786.
```

Reply:
I don't think this makes sense

Reply:
... because this has nothing to do with Envoy proxy. Its part of pilot code

Reply:
then I don't understand what it is for. :). What exactly is the matching?  I had thought it is matching to apply the filter to the right proxies.